docs/content.go

```diff
@@ -715,10 +715,39 @@ Enterprise Performance Computing (EPC)`
 	InspectUse   string = `inspect [inspect options...] <image path>`
 	InspectShort string = `Show metadata for an image`
 	InspectLong  string = `
-  Inspect will show you labels, environment variables, and scripts associated 
-  with the image determined by the flags you pass.`
+  Inspect will show you labels, environment variables, apps and scripts associated 
+  with the image determined by the flags you pass.
+  `
 	InspectExample string = `
-  $ singularity inspect ubuntu.sif`
+  $ singularity inspect ubuntu.sif
+  
+  If you want to list the applications (apps) installed in a container (located at
+  /scif/apps) you should run inspect command with --list-apps <container-image> flag.
+  ( See https://sci-f.github.io for more information on SCIF apps)
+
+  The following environment variables are available to you when called 
+  from the shell inside the container. The top variables are relevant 
+  to the active app (--app <app>) and the bottom available for all 
+  apps regardless of the active app:
+  ACTIVE APP ENVIRONMENT:
```

Comment:
I would add something to indicate that the user can interact with these also during build / development. Maybe something like:

> Both sets of variables are also available during development, meaning when you build the container.

Reply:
And double check this was honored with the transition to golang - it could be that it's missing (and we should open an issue). The standalone scif client (both python and golang) make these available during build so the user doesn't need to remember `/scif/apps/<appname>` etc.